docker-compose.yml

```diff
@@ -57,6 +56,9 @@ services:
       # This should correspond to the server declared in PHPStorm `Preferences | Languages & Frameworks | PHP | Servers`
       # Then PHPStorm will use the corresponding path mappings
       PHP_IDE_CONFIG: serverName=localhost
+      CORS_ALLOW_ORIGIN: "*"
+      API_DOMAIN: ".*"
+      SHARED_COOKIE_DOMAIN: ""
```

Comment:
we have to discuss how we want to handle the security for the development environment